include/manifold/conv.h

```diff
@@ -17,18 +17,24 @@
 #include <vector>
 
 #include "manifold/common.h"
+#ifdef MANIFOLD_CROSS_SECTION
 #include "manifold/cross_section.h"
+#endif
 #include "manifold/manifold.h"
 #include "manifold/types.h"
 
 using namespace manifold;
 using ManifoldVec = std::vector<Manifold>;
+#ifdef MANIFOLD_CROSS_SECTION
```

Comment:
I think we can skip this if we continue to say the bindings always require Clipper2. Or was someone wanting C bindings without cross-section?

Reply:
I did that because I'm always building the C bindings as part of the main libmanifold, so they need to conditionalize based on whether the user designates cross_section as on or off.  It might be a little cleaner to break the C binding logic specific to cross_section out into a separate file that can be turned on or off in the build logic, if you prefer.

Reply:
That sounds like a good idea, but I'll defer to your judgement.

Reply:
I played with it a little, and based on that for the moment I'd prefer to leave things as is, since it's compiling and working.  We can always refine it later.